tests/import_test.go

```diff
@@ -1104,4 +1124,45 @@ var _ = Describe("Preallocation", func() {
 			return utils.NewDataVolumeForBlankRawImageBlock("import-dv", "100Mi", f.BlockSCName)
 		}),
 	)
+
+	It("Filesystem overhead is honored with blank volume", func() {
+		defaultSCName := utils.DefaultStorageClass.GetName()
+		testedFilesystemOverhead := &cdiv1.FilesystemOverhead{}
+		globalOverhead := "0.055"
+		testedFilesystemOverhead.Global = cdiv1.Percent(globalOverhead)
+
+		config.Spec.FilesystemOverhead = testedFilesystemOverhead.DeepCopy()
+		By(fmt.Sprintf("Updating CDIConfig filesystem overhead to %v", config.Spec.FilesystemOverhead))
+		err := utils.UpdateCDIConfig(f.CrClient, func(config *cdiv1.CDIConfigSpec) {
+			config.FilesystemOverhead = testedFilesystemOverhead.DeepCopy()
```

Comment:
Isn't it clearer to use config.Spec.FilesystemOverhead.DeepCopy() here?

Reply:
Since I used the code from other test, I refactored the test to avoid duplication.